docs/docs/tracing/production.mdx

```diff
@@ -0,0 +1,70 @@
+---
+sidebar_position: 6
+sidebar_label: Production Monitoring
+---
+
+# Tracing in Production
+
+Machine learning projects don't conclude with their initial launch. Ongoing monitoring and incremental enhancements are critical for long-term success. [MLflow Tracing](..) offers observability for your production application, supporting the iterative process of continuous improvement.
+
+When it comes to production monitoring, there are many additional considerations for tracing, such as scalability, security, and cost. Moreover, **monitoring the machine learning model alone never be sufficient in the real software operation**. Engineering teams need to monitor multiple services throughout the entire product and ensure that it delivers expected values to end users. Thereby, MLflow supports integration with various observability solutions such as Grafana, Datadog, New Relic, and more, with the power of OpenTelemetry standardization.
+
+
+## Self-host Tracking Server
+
+Of course, you can keep using the MLflow tracking server to store production traces. However, tracking server is optimized for offline experience and generally not suitable for handling the hyper scale traffic. Thereby, we recommend using the other two options for production monitoring use case.
+
+If you choose to keep using the tracking server in production, we **strongly recommend** using SQL-based tracking server on top of a scalable database and artifact storage, as it will be a key factor for write and query performance. Refer to the [tracking server setup guide](../../tracking/#tracking-setup) for more details. In addition, tracking server by default uses infinite retention date for trace data, hence it is recommended to set up periodical deletion job using the SDK or REST API.
+
+
+## OpenTelemetry Collector
+
+Traces generated by MLflow are compatible with the [OpenTelemetry trace specs](https://opentelemetry.io/docs/specs/otel/trace/api/#span).
+Therefore, MLflow traces can be exported to various observability solutions that support OpenTelemetry.
+
+// TODO: Add list of icons
```

Comment:
note: Planning to add logo tiles similar to the library integrations, with actual examples of exporting traces to those 3P services.

Reply:
@B-Step62  Not sure if this is intended or not but this line isn't a comment so it appears in the page. Comments in MDX use this syntax:

```
{/* ... */}
```